File: 217-ContainsDuplicate/Solution.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        visited = {}
        for value in nums:
            if(value in visited):
                return True
#Check if the value exists in the dictionary
            else:
                visited[value] = 1
#input a placeholder value to allow for constant look up time
        return False

File: 217-ContainsDuplicate/Solution.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashSet<Integer> visited = new HashSet<Integer>();
        for (int val : nums){
            if(visited.contains(val)){
                return true;
            }
            else{
                visited.add(val);
            }
        }
        return false;
    }
}
